### PR TITLE
brightening up some bright theme colors

### PR DESCRIPTION
**Pull request by Harvtronix**
- brighter comments
- brighter editor background
- brighter diff editor highlighting
- brighter and grayer selection background
- bright tab (one with most contrast) is now the selected one

### CODE REVIEW

**Review by Tyriar — COMMENTED** [review]
The lighter and dim variants intentionally kept the same colors for everything except the background, what's the rationale here and could you post some screenshots of the results?

**Comment by Harvtronix** [comment]
No worries! If you'd prefer to keep them identical, I can continue using my fork locally or publish an off-shoot of the sapphire bright theme (under MIT as well and with appropriate credit, of course). 

Sapphire light is my favorite theme, but I have trouble seeing some of the colors I use frequently due to their low contrast, so I have them adjusted locally and thought maybe others could benefit as well.

In particular, the text selection and diff editor are very low contrast, so locally I have them bumped up for accessibility reasons. Same thing with comments. I want them to be subtle, but the default is a bit too subtle for my vision.

Also, the dark tab being the selected one was really throwing me off, so I switched to have the active tab be brighter and the others blending more into the tab bar background.

Here's some pics of the changes:

## Comments and editor background
old:
image

new:
image

## Diff editor
old:
image

new: 
image

## Selection background
old:
image

new:
image

## Tab colors
old:
image

new:
image

File: index.ts [hunk 1]
```diff
@@ -94,8 +94,39 @@ const colorSet: IColorSet = {
...
+
+    'diffEditor.insertedTextBackground': '#0f02',
+    'diffEditor.removedTextBackground': '#f003',
+    'diffEditor.insertedTextBorder': '#0004',
+    'diffEditor.removedTextBorder': '#0004',
+
+    'editor.background': '#2a3447',
+    'editor.selectionBackground': '#fff3',
+
+    'editorWhitespace.foreground': '#fff2',
```
Comment (Harvtronix) [inline index.ts]: Ignore this change. I need to remove it and rebuild the theme. This should be a local-to-me change only.

**Review by Tyriar — APPROVED** [review]
Looks better 👍, once you remove that change should be good to go. FYI if you didn't know, you can add theme-specific user overrides like this:

```
  "workbench.colorCustomizations": {
    "[Sapphire (Bright)]": {
      "editorWhitespace.foreground": "#fff2"
    }
  }
```

**Comment by Harvtronix** [comment]
Sorry for the delay in updating this PR! Thanks for the tip. I think I must have copied over all of my local changes to the theme and forgot to remove that one 😅 

~I may have another PR coming your way for your consideration to colorize the diff lines in the editor hover widget.~ Looks like the theme generator doesn't currently support token color overrides, so i might not be able to make that change. It would require setting `markup.inserted` and `markup.deleted`.

**Review by Tyriar — APPROVED** [review]
Thanks @Harvtronix. Sorry about the delay, been a bit swamped.

**Comment by Tyriar** [comment]
Published v0.9.3

**Pull request merged**